### PR TITLE
feat(cli): add `docs` command

### PR DESCRIPTION
**Pull request by johnyob**
# Context

# Description

**Dependencies**: #365 

This PR adds the `docs` command to the `jstz` CLI, which opens a browser page (from the terminal) to the `jstz` docs (hosted on github pages)

# Manually testing the PR

```
cargo run --bin jstz -- docs
```

### CODE REVIEW

File: crates/jstz_cli/src/term/browser.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        Command::new("sh")
+            .arg("-c")
+            .arg(&mac_cmd)
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
+            .spawn()?;
+    };
+
+    info!(
+        "Opened a link in your default browser: {}",
```
Comment (alanmarkoTrilitech) [inline crates/jstz_cli/src/term/browser.rs:28]: this is just a log, not something that gets printed in terminal, right?

**Review by alanmarkoTrilitech — APPROVED** [review]

File: crates/jstz_cli/src/term/browser.rs (hunk 1 above)
Comment (johnyob) [reply crates/jstz_cli/src/term/browser.rs:28]: No, this gets printed in the terminal

Comment (johnyob) [reply crates/jstz_cli/src/term/browser.rs:28]: Like:
```sh
$ jstz docs
Opened a link in your default browser: https://trilitech.github.io/jstz/
```

Comment (alanmarkoTrilitech) [reply crates/jstz_cli/src/term/browser.rs:28]: oh ok. I would prefer more not printing anything or atleast having a slightly simpler message e.g.:
"Opened docs - https://trilitech.github.io/jstz/ in your default browser."

Comment (johnyob) [reply crates/jstz_cli/src/term/browser.rs:28]: Modulo a 'docs', the messages are basically identical. The message is more generic to permit different links being opened in the future (e.g. wallets)

**Pull request merged**